Libraries/ART/ReactNativeART.js

```diff
@@ -439,11 +439,12 @@ function extractFont(font) {
   }
   var fontFamily = extractSingleFontFamily(font.fontFamily);
   var fontSize = +font.fontSize || 12;
+  var fontWeight = font.fontWeight ? font.fontWeight.toString() : '400';
   return {
```

Comment:
If i set fontWeight to `0`, it will fallback to `400`, so i think it's better using `font.fontWeight != null ? font.fontWeight.toString() : '400'`